### DISCUSSION TITLE
🎭 | Locales

### DISCUSSION THREAD

**Discussion by EhsanFox** [SeTar-Bot/Setar-Bot#62] [Q&A] [answered]
### [Questions related to Locales](https://github.com/SeTar-Bot/bot/tree/main/src/locales)

👀 Make sure to tell us:
- Are you following or changing anything in [Locale schema](https://github.com/SeTar-Bot/bot/blob/main/types/localeSchema.ts)
- What branch/version you are using
- Are you changing or creating a new locale
- Are you adding any new method that doesn't exist on [Locale Schemas (Reply or Error)](https://github.com/SeTar-Bot/bot/blob/main/types/localeSchema.ts)
- Have you added your locale to the [List](https://github.com/SeTar-Bot/bot/blob/main/src/locales/list.ts) and [Enum list](https://github.com/SeTar-Bot/bot/blob/main/src/typings/enums.ts#L41)

Also, don't forget to [Join our Discord Community](https://discord.gg/7jgfP6j4Tc)

**Comment by im-parsa** [comment, marked as answer]
Hey @EhsanFox 👋🏻 ,

My idea is to hire several people for translation so that we can make the community bigger and more famous.

**Reply by EhsanFox** [reply]
This is a nice idea, but if is there any people available to do it for us and for the community it would be even nicer. we are all looking for everyone who can help us with this project 😄
